crates/rolldown/src/stages/link_stage/bind_imports_and_exports.rs

```diff
@@ -178,6 +179,16 @@ impl LinkStage<'_> {
     self.errors.extend(binding_ctx.errors);
     self.warnings.extend(binding_ctx.warnings);
 
```

Comment:
Can not share the same struct with `NamedExport` merger, since https://github.com/rolldown/rolldown/blob/1cba71aedca0cfc34cb0dcc3ac917ed92d0c7563/crates/rolldown_common/src/types/named_import.rs?plain=1#L19 for namespace named import the **imported** is meanless